utils/utils_test.go

```diff
@@ -35,18 +35,28 @@ func TestHexToPubkey(t *testing.T) {
 	_, err := HexToPubkey("0x01")
 	require.Error(t, err)
 
-	a, err := HexToPubkey("0xed7f862045422bd51ba732730ce993c94d2545e5db1112102026343904fcdf6f5cf37926a3688444703772ed80fa223f")
+	// This is an invalid pubkey
+	_, err = HexToPubkey("0xed7f862045422bd51ba732730ce993c94d2545e5db1112102026343904fcdf6f5cf37926a3688444703772ed80fa223f")
```

Comment:
non blocking: can be written as a table driven test to separate the test cases

Reply:
Okay yeah good suggestion. I'll update this. And fix the linter issue :(